src/hotspot/share/gc/shenandoah/shenandoahFreeSet.cpp

```diff
@@ -24,6 +25,7 @@
 
 #include "precompiled.hpp"
 #include "gc/shared/tlab_globals.hpp"
+#include "gc/shenandoah/shenandoahBarrierSet.hpp"
```

Comment:
Do we really need to include `shenandoahBarrierSet.hpp` here?

Reply:
Thanks.  Removed.